packages/react-core/src/components/Card/Card.tsx

```diff
@@ -64,7 +64,7 @@ export const CardContext = React.createContext<Partial<CardContextProps>>({
 });
 
 export const Card: React.FunctionComponent<CardProps> = ({
-  children = null,
+  children,
   id = '',
   className = '',
```

Comment:
We could remove the default value across the Card sub-components for props like `id` and `className`